stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/MessagesActivity.kt

```diff
@@ -139,26 +142,45 @@ class MessagesActivity : AppCompatActivity() {
                 )
             }
 
-            if (selectedMessage != null) {
-                SelectedMessageMenu(
-                    modifier = Modifier
-                        .align(Alignment.Center)
-                        .padding(horizontal = 20.dp)
-                        .wrapContentSize(),
-                    shape = ChatTheme.shapes.attachment,
-                    messageOptions = defaultMessageOptionsState(
-                        selectedMessage = selectedMessage,
+            if (selectedMessageState != null) {
+                val selectedMessage = selectedMessageState.message
+                if (selectedMessageState is SelectedMessageOptionsState) {
+                    SelectedMessageMenu(
+                        modifier = Modifier
+                            .align(Alignment.Center)
+                            .padding(horizontal = 20.dp)
+                            .wrapContentSize(),
+                        shape = ChatTheme.shapes.attachment,
+                        messageOptions = defaultMessageOptionsState(
+                            selectedMessage = selectedMessage,
+                            currentUser = user,
+                            isInThread = listViewModel.isInThread
+                        ),
+                        message = selectedMessage,
+                        onMessageAction = { action ->
+                            composerViewModel.performMessageAction(action)
+                            listViewModel.performMessageAction(action)
+                            listViewModel.removeOverlay()
+                        },
+                        onDismiss = { listViewModel.removeOverlay() }
+                    )
+                } else if (selectedMessageState is SelectedMessageReactionsState) {
+                    SelectedReactionsMenu(
+                        modifier = Modifier
+                            .align(Alignment.Center)
+                            .padding(horizontal = 20.dp)
+                            .wrapContentSize(),
+                        shape = ChatTheme.shapes.attachment,
+                        message = selectedMessage,
                         currentUser = user,
-                        isInThread = listViewModel.isInThread
-                    ),
-                    message = selectedMessage,
-                    onMessageAction = { action ->
-                        composerViewModel.performMessageAction(action)
-                        listViewModel.performMessageAction(action)
-                        listViewModel.removeOverlay()
-                    },
-                    onDismiss = { listViewModel.removeOverlay() }
-                )
+                        onMessageAction = { action ->
+                            composerViewModel.performMessageAction(action)
+                            listViewModel.performMessageAction(action)
+                            listViewModel.removeOverlay()
```

Comment:
Same as [this](https://github.com/GetStream/stream-chat-android/pull/2782/files#r772982932) comment.